tutorials/text/text_props.py

```diff
@@ -194,7 +194,15 @@
 # +------------------------------------------+--------------------------------+
 #
 #
-# which are lists of font names.
+# If any of generic family names appear in ``'font.family'``, we replace that entry
+# by all the entries in the corresponding rcParam mapping.
+# For example: ::
+#
+#    matplotlib.rcParams['font.family'] = ['MyFamily1', 'serif', 'MyFamily2']
```

Comment:
What is `MyFamily1` and `MyFamily2`?  Are these fonts or font families?  How do I get these names?

Reply:
These are font families, like I said in the previous comment, it can be anything other than those 5 special keywords.

Reply:
So I'd change these to `Family1` or `FontFamily1`, or even just be specific with generic fonts...

Also, what do these default to?  What are the default 'serif' fonts?

Reply:
The default "generic-family" fonts default to the ones in rcParams, mentioned in the [docs](https://matplotlib.org/stable/tutorials/introductory/customizing.html):
![image](https://user-images.githubusercontent.com/43996118/121299410-01ff4400-c913-11eb-9bea-6262aa9c50c9.png)